src/converter.rs

```diff
@@ -135,7 +135,34 @@ fn convert_line(line: &str) -> String {
 
         let (mut windows_command, flags_mappings, additional_arguments, modify_path_separator) =
             match shell_command {
-                "cp" => ("xcopy".to_string(), vec![("-[rR]", "/E")], vec![], true),
+                "cp" => {
+                    // There is no good `cp` equivalent on windows. There are
+                    // two tools we can rely on:
+                    //
+                    // - xcopy, which is great for directory to directory
+                    //   copies.
+                    // - copy, which is great for file to file/directory copies.
+                    //
+                    // We can select which one to use based on the presence of
+                    // the -r flag.
+                    let win_cmd = match Regex::new("-[^ ]*[rR]") {
+                        Ok(regex_instance) => {
+                            if regex_instance.is_match(&arguments) {
+                                "xcopy".to_string()
+                            } else {
+                                "copy".to_string()
+                            }
+                        }
+                        Err(_) => "del".to_string(),
```

Comment:
if regex fails, we start deleting files :)
why not default to xcopy?

Reply:
lol, copy pasta fail.

Reply:
On the other hand, why do we fail this way? This should probably be an `expect` and panic - it indicates something is statically wrong in the code.

Reply:
maybe... still... i feel as a text converter, maybe we can just default to something if we have an issue.

Reply:
I changed the two regexes to panic - which I believe is the better behavior here. If you don't want that, I'll revert and put the correct default :).